l10n/en/navigation_v2.ftl

```diff
@@ -5,7 +5,7 @@
 
 navigation-v2-mozilla = { -brand-name-mozilla }
 navigation-v2-download-firefox = Download { -brand-name-firefox }
-navigation-v2-get-a-firefox-account = Get a { -brand-name-firefox-account }
+navigation-v2-get-mozilla-vpn = Get { -brand-name-mozilla-vpn }
```

Comment:
check if this should be a replacement for account translation or an additional string

Reply:
Keep account translation and label obsolete